src/forms/inputs/radio-group.js

```diff
@@ -89,9 +90,18 @@ const defaultProps = {
   radioInputProps: {},
 }
 
-function RadioGroupLegend({ label, name }) {
-  if (label === false) return null
-  return <legend>{label || convertNameToLabel(name)}</legend>
+function RadioGroupLegend({ label, name, required, requiredIndicator, hint }) {
```

Comment:
Since this is the same for the checkbox group, would it make sense to pull this out into it's own component (just a `legend` component)? That way it could also be used separately on projects if needed.

Reply:
Good idea. I'm happy to pull that out into a separate issue/PR if it helps with timing (wouldn't want that to happen without new tests, documentation, etc.)